europy/lifecycle/report.py

```diff
@@ -41,8 +43,23 @@ def __init__(self, title: str = "EuroPy Test Report"):
 
         self.timestamp = datetime.now()
 
+    def to_markdown(self):
+        markdownReport = Markdown()
+        markdownReport.add_header(self.title)
+        markdownReport.add_header('Model Information')
+        markdownReport.add_dict_content(self.model_card['details'].__dict__)
+        markdownReport.add_dict_content(self.model_card['parameters'])
+        markdownReport.add_horizontal_line()
+        markdownReport.add_header('Test Results')
+        markdownReport.add_horizontal_line()
+        markdownReport.add_dict_content(json.loads(json.dumps(self.test_results, cls=Encoder)))
```

Comment:
I think for the test results we will need to be more specific here and define a `to_markdown` function in the TestResult class and loop over it here to add it to the report. This is because we want to control how things like data frames are displayed (right now it is just a list of lists, when we want to call the data frame to_markdown method which gives us a nice markdown table).